cmd/soroban-rpc/internal/xdr2json/conversion.go

```diff
@@ -0,0 +1,144 @@
+//nolint:lll
```

Comment:
What are the offending lines? Are they unavoidably long? Should we, otherwise tolerate longer lines in general (which is totally file, but I would rather do that than sprinkle files with `nolint:lll`)?

Reply:
It's the C compilation lines; you had the same directive in preflight.go